### ISSUE TITLE
SureSites Pallet: Handle Bad Data

### ISSUE THREAD

**Issue by stdavis** [agrc/locate#178]
Forklift is having issues with the data:
```
DEBUG   03-25 11:40:49 SureSitesPallet:  100 utahsuresites.com receieved in 4068 ms with response code 200
DEBUG   03-25 11:40:50 SureSitesPallet:  115 processing site: 1376
WARNING 03-25 11:41:07 SureSitesPallet:  205 'NoneType' object is not subscriptable
INFO    03-25 11:41:07 SureSitesPallet:  206 attempting to use coords from "Position" field
ERROR   03-25 11:41:07       lift:  134 error processing pallet: Expecting value: line 1 column 1 (char 0) for pallet: c:\forklift\warehouse\locate\scripts\SureSitesPallet.py:SureSitePallet
Traceback (most recent call last):
  File "c:\program files\arcgis\pro\bin\python\envs\forklift\lib\site-packages\forklift\lift.py", line 129, in process_pallets
    pallet.process()
  File "c:\forklift\warehouse\locate\scripts\SureSitesPallet.py", line 117, in process
    row = self._map_site_to_row(site, json_properties, geocoder)
  File "c:\forklift\warehouse\locate\scripts\SureSitesPallet.py", line 209, in _map_site_to_row
    geojson = loads(site['Position'])['coordinates']
  File "c:\program files\arcgis\pro\bin\python\envs\forklift\lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "c:\program files\arcgis\pro\bin\python\envs\forklift\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "c:\program files\arcgis\pro\bin\python\envs\forklift\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

Perhaps they changed a property name or something?

**Comment by nathankota** [comment]
I'll check in with them.

**Comment by stdavis** [comment]
Fixed via: 15b523444c2a2f5c2c90cd51198a2edc0b993796
